src/Web/Avalonia.Web.Blazor/AvaloniaView.razor.cs

```diff
@@ -299,6 +300,12 @@ protected override void OnAfterRender(bool firstRender)
                     _topLevel.Prepare();
 
                     _topLevel.Renderer.Start();
+
+                    if (_topLevel.Renderer is DeferredRenderer dr)
```

Comment:
Note: this is technically a hack, but it's a kinda unique use case when
- we want to blit the previous frame
- renderer doesn't have much control over the render target
- we render on the UI thread

We also don't want to have it as a meaningful public API.

Therefore we have InternalsVisibleTo hack here.

Reply:
This type of thing really needs to be commented in-code for future contributors.